src/guide/introduction.md

```diff
@@ -1,5 +1,5 @@
 ---
-footer: false
+footer: true
```

Comment:
**question**: could you please provide a little bit more context on why this change has been made?

Reply:
- [ ]